portal/assets/js/global.js

```diff
@@ -35,4 +68,20 @@ document.addEventListener('DOMContentLoaded', function() {
     addMaildrop.parentNode.insertBefore(inputDropClone, addMaildrop);
   });
 
+
+  // Get user's infos
+  var r = new XMLHttpRequest();
+  r.open("GET", "/ynhpanel.json", true);
```

Comment:
Eh, where is that supposed to come from ? x_x

Reply:
Hm now that I dig a bit deeper I kinda understand, but while testing on my machine, before logging in, it can't access this file, and after logging in there's no 'theme' property yet defined apparently :s

Reply:
Well this comes initially from the ynhpanel.js file, it was used to fetch config and is not necessary there anymore. I can't say I really understand the organization behind the serving of the config, so I decided to use the same system that was used and worked.

Reply:
> there's no 'theme' property yet defined apparently

ah yes I guess if you've never defined the theme from the command line with the other PR in yunohost-admin it will not be defined I guess

Reply:
> before logging in

I didn't test pre-logging, true. But yes it would be nice that the theme extends to the login page. Maybe we can figure out another way to pass the chosen theme setting, but this is the part of code that I understand less.